tests/e2e/logging/test_openshift-logging.py

```diff
@@ -88,10 +104,36 @@ def create_instance():
     logger.info(f'The installed CSV is {get_csv}')
 
 
-def teardown():
+def teardown(update_images):
     """
     The teardown will uninstall the openshift-logging from the cluster
     """
+
+    if not update_images:
+        cur_dir = os.getcwd()
+        file_list = ['Image_Mapping.txt', 'mirror_docker.conf', 'mirror_result.txt',
+                     'OperatorSource_CSV_Files.txt', 'OperatorSource_Images_List.txt',
+                     'OperatorSource_Images_registry_proxy.txt',
+                     'OperatorSource_Images_version.txt', 'quay.token']
+        dir_list = [
```

Comment:
are  you considering to have list more than 1 item  later?, i see you loop threw the list  later  in line 128, where but the list just contains one item there.